packages/frontend/config/environment.js

```diff
@@ -79,11 +79,11 @@ module.exports = function (environment) {
   };
 
   if (environment === 'development') {
-    // ENV.APP.LOG_RESOLVER = true;
-    // ENV.APP.LOG_ACTIVE_GENERATION = true;
-    // ENV.APP.LOG_TRANSITIONS = true;
-    // ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
-    // ENV.APP.LOG_VIEW_LOOKUPS = true;
+    ENV.APP.LOG_RESOLVER = process.env.LOG_RESOLVER || false;
```

Comment:
since code climate is being a jerk about it, could you try the following? should be functionally equivalent.

```suggestion
    ENV.APP.LOG_RESOLVER = !!process.env.LOG_RESOLVER;
```